src/map/mob.cpp

```diff
@@ -2415,6 +2415,7 @@ void mob_damage(struct mob_data *md, struct block_list *src, int damage)
 	}
 
 	if (src && damage > 0) { //Store total damage...
+		mobskill_use(md, gettick(), (skill_id<<16), damage);
```

Comment:
Maybe the call could be instead from https://github.com/rathena/rathena/blob/97b5f3b4ac63bb920a673043af02518784c386f5/src/map/battle.cpp#L284

Reply:
Ground/Unit skill damage doesn't use battle damage, that's why part isn't correct either like reported here:
https://github.com/rathena/rathena/issues/7141